### ISSUE TITLE
Add 2018 BLS data

### ISSUE THREAD

**Issue by ptatian** [NeighborhoodInfoDC/BLS#5]
Add 2018 BLS county data to Bls.Bls_allgeos_country and Bls.Bls_county_was15 data sets. 

Data download from https://www.bls.gov/cew/datatoc.htm. NAICS based - all industries, CSVs by area (annual averages) - has separate CSVs for counties and metro areas.  or if look at the county high-level file. Indicator we want to look at is: Annual Average Employment.

**Comment by ptatian** [comment]
- [ ] Download and convert 2018 CSV file to SAS using STAT Transfer.
- [ ] Add 2018 county data set to **prog\Read BLS.sas**.
- [ ] Update **prog\Read BLS.sas** header to include description of data source (including download URL) and STAT Transfer conversion process. Explanation in header should be sufficient for someone who needs to repeat these steps in future years. 
- [ ] Replace nonstandard space characters in variable labels with standard blank space. (See L:/Metadata/meta_bls_bls_allgeos_country.html for places where �'s appear.)

**Comment by rpitingolo** [comment]
@ptatian is it worth adding 2018 to the housing market analysis if I'm able  to add it to the warehouse today or tomorrow?

**Comment by rpitingolo** [comment]
@ptatian actually, it looks like 2018 is only available on the Quarterly file, not the Annual Averages.

**Comment by ptatian** [comment]
@rpitingolo Yes, I noticed that, too. It is hard to tell but [seems like the annual data should be available soon](https://www.bls.gov/sae/790sced.htm). We should keep checking.

**Comment by rpitingolo** [comment]
@ptatian OK. I wonder if the shutdown had an impact on the data schedule? I'll keep an eye on it.
